.ci_tools/before_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Handler for before_install step in CI
+# Functionalities:
+# 1.  install sdkmanager manually
+# 2.  TODO: install whole new SDK
+# 3.  TODO: point sdk location to our installed one ( not the travis's )
+# Side Effect:
+#     $CI_SDK_DIR is left behind on the disk: could not delete.
+
+set -e
+
+CI_SDK_DIR=$HOME/ci_sdk
+SDK_TOOL_FILENAME=sdk-tools-linux-4333796.zip
```

Comment:
It looks like the script now has version dependencies. So we will have to update them when the SDK is updated.